src/main/java/ua/bookstore/online/exception/CustomGlobalExceptionHandler.java

```diff
@@ -56,6 +58,11 @@ protected ResponseEntity<ErrorResponseDto> handleMethodNotFound(EntityNotFoundEx
         return getResponseEntity(NOT_FOUND, ex.getMessage());
     }
 
+    @ExceptionHandler(AccessDeniedException.class)
+    protected ResponseEntity<ErrorResponseDto> handleMethodNotFound(AccessDeniedException ex) {
+        return getResponseEntity(FORBIDDEN, ex.getMessage());
+    }
```

Comment:
```suggestion
    protected ResponseEntity<ErrorResponseDto> handleAccessDenied(AccessDeniedException ex) {
        return getResponseEntity(FORBIDDEN, ex.getMessage());
    }
```
A better name for the method could be handleAccessDenied or handleForbidden, because might be a bit misleading as it suggests a "method not found" error, while the method actually handles an AccessDeniedException

Reply:
Copy-past in action)